views/forms/forms_view.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * Displays all forms belonging to a specific participant.
+ * Links to view, edit, or delete those forms are available.
+ *
+ * @author Michelle Crawley
+ * @copyright 2017 Marist College
+ * @version 0.7.1
+ * @since 0.1
+ */
+
+global $db, $params, $route;
+$peopleid = $params[0];
+
+# Get participant first name, last name, middle
+$result = $db->query("SELECT firstname, lastname, middleinit FROM people WHERE peopleid = $1", [$peopleid]);
+$name = pg_fetch_assoc($result);
+extract($name);
+
+# Displays the participant's name in the title according to whether they have a middle name
+$route['title'] = "Forms Completed for " . ucwords($firstname . " " . ($middleinit ?: "") . " " . $lastname);
```

Comment:
Doesn't this give them `""` if they do actually have a middleInit? 
```
$v = 'My Value';
$r = ($v) ?: 'No Value'; // $r is set to 'My Value' because $v is evaluated to TRUE
```

Reply:
I'm pretty sure this works. Also you're example explains why it works I believe

Reply:
You're right. I spoke too quick =(